tests/ViewHelpers/PropertyViewHelperTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+require('vendor/autoload.php');
+
+use ArmadilloEstates\ViewHelpers\PropertyViewHelper;
+
+class PropertyViewHelperTest extends PHPUnit\Framework\TestCase
+{
+    public function testSuccessDisplayAll()
+    {
+        $entity = $this->createMock(\ArmadilloEstates\Entities\PropertyEntity::class);
+        $entity->expects($this->once())
+            ->method('getAddress1')
+            ->willReturn('Address1');
+
+        $entity->expects($this->once())
+            ->method('getAddress2')
+            ->willReturn('Address2');
+
+        $entity->expects($this->once())
+            ->method('getTown')
+            ->willReturn('Town');
+
+        $entity->expects($this->once())
+            ->method('getTypeName')
+            ->willReturn('Sale');
+
+        $entity->expects($this->once())
+            ->method('getStatusName')
+            ->willReturn('Sold');
+
+        $entity->expects($this->once())
+            ->method('getImage')
+            ->willReturn('Image');
+
+        $collection = $this->createMock(\ArmadilloEstates\Collections\PropertyCollection::class);
+        $collection->expects($this->once())
+            ->method('getAllProperties')
+            ->willReturn([$entity]);
+
+        $expected = '<div class="col-4 text-center mb-4"><div class="card"><h5 class="card-title pt-3">Address1 Address2<br>Town</h5><img class="card-img-top cardImg" src=Image alt="Card image cap"><div class="card-body"><p class="card-text">Sale</p><p class="card-text">Sold</p><a href="#" class="btn btn-primary">View More...</a></div></div></div>';
```

Comment:
Could you format this so it's more readable?

Reply:
no. the string needs to be on one line.